include/seqan/file/string_external.h

```diff
@@ -94,7 +94,7 @@ namespace seqan
  *
  * @section Remarks
  *
- * When using this configuration ,th eSize type o fthe ExternalString is Size type of <tt>TFile</tt>.  Normally, this is
+ * When using this configuration ,the Size type of the ExternalString is Size type of <tt>TFile</tt>.  Normally, this is
```

Comment:
highlight Size if referring to meta-function. Like wrap in @link @endlink or in \<tt\> \</tt\>